### ISSUE TITLE
use TextGeometry set high curveSegments has holes that not smooth

### ISSUE THREAD

**Issue by honlyHuang** [mrdoob/three.js#25950]
### Description

I set curveSegments 36， the text like this
<img width="755" alt="image" src="https://user-images.githubusercontent.com/3080820/235101351-50d1870f-409d-4ff4-bf27-184b280d0f53.png">

use `BufferGeometryUtils.mergeVertices` , it smooth 

<img width="789" alt="image" src="https://user-images.githubusercontent.com/3080820/235101770-707c7b48-954c-4a43-b9bf-be9730446535.png">

I have a question,  may put merge vertices in the ExtrudeGeometry when generate vertices。

### Reproduction steps

1. loader a text 
2. set curveSegments 36 or more high
3. 

### Code

```js
let textGeo: BufferGeometry = new TextGeometry(text, {
        font,
        size: 2,
        height: 20,
        curveSegments: 36,
        bevelEnabled: true,
        bevelThickness: 0.1,
        bevelSize: 0.1,
        bevelOffset: 0,
        bevelSegments: 20
      })
```

### Live example

* [jsfiddle-latest-release](https://jsfiddle.net/g3atw6k5/)
* [jsfiddle-dev](https://jsfiddle.net/hjqw94c5/)

### Version

139

### Device

Desktop

### Browser

Chrome

### OS

MacOS

**Comment by Mugen87** [comment]
> set curveSegments 36 or more high

I can't reproduce with the following live example. I've used the code from [webgl_geometry_text](https://threejs.org/examples/webgl_geometry_text) and just change the `curveSegments` from `4` to `36`. The text mesh renders as expected.

https://jsfiddle.net/b8ptozrm/1/

Do you mind updating the fiddle to demonstrate the issue?

**Comment by Mugen87** [comment]
Closing. The issue can be reopened when the above fiddle is updated so it shows the glitch.

**Comment by honlyHuang** [comment]
@Mugen87  like this https://jsfiddle.net/1kLyc6qj/

```
textGeo = new TextGeometry( text, {

    font: font,

    size: 150,
    height: height,
    curveSegments: 36,

    bevelThickness: 0.1,
    bevelSize: 0.1,
    bevelOffset: 0,
    bevelSegments: 20,
    bevelEnabled: bevelEnabled

  } );
```

<img width="308" alt="image" src="https://user-images.githubusercontent.com/3080820/236395838-44f66825-3a1a-46f1-befe-32702321e0aa.png">

**Comment by Mugen87** [comment]
Still don't see the glitches from your screenshot. On my macMini, the fiddle using `mergeVertices()` looks like so:

<img width="863" alt="image" src="https://user-images.githubusercontent.com/12612165/236405205-f78bd1f4-1fa5-4866-ad3b-e89adb8f91f9.png">

Can you please test with another computer to verify if this is a device specific issue?

**Comment by honlyHuang** [comment]
You're right, it seems to be related to the device. I replaced the computer and there was no such problem. 
My mac
<img width="586" alt="image" src="https://user-images.githubusercontent.com/3080820/236591844-0d61be8d-1639-4c8b-b2bd-adee66f1884e.png">

**Comment by Mugen87** [comment]
Is it possible to upgrade your mac to a higher version?
